src/jit/emitarm64.cpp

```diff
@@ -3266,31 +3271,31 @@ void                emitter::emitIns_R(instruction ins,
     case INS_br:
         assert(isGeneralRegister(reg));
         id = emitNewInstrCns(attr, 0);
-        fmt = IF_BR_1A;
+        assert(id != nullptr);
```

Comment:
I think that the assert can be removed.
As we don't do this anyplace else and we can be fairly sure that it won't be null as we just assigned it.